### ISSUE TITLE
Design beautiful report letterheads

### ISSUE THREAD

**Issue by jniles** [IMA-WorldHealth/bhima#869]
PDF reports seem to be working well; however, they are rather dull without a nice letterhead.  To really impress our users, we should design a pretty letterhead with the hospital logo, the IMA logo, and some of the default enterprise information that is useful for any generic report.

It would be ideal if this were able to be included easily from the report configuration page.  That way, our users can optionally choose to remove it to save on printer ink.

**Comment by DedrickEnc** [comment]
You are right about the letterhead, we should use it, but I am not understanding your last paragraph.
